ecosystem/runners/runner.py

```diff
@@ -43,6 +47,39 @@ def tear_down(self):
         if self.cloned_repo_directory and os.path.exists(self.cloned_repo_directory):
             shutil.rmtree(self.cloned_repo_directory)
 
+    def get_config(self, file, file_fault, ecosystem_deps: Optional = None):
+        """check for configuration file"""
+        if self.repo_config is not None:
+            repo_config = self.repo_config
+        elif os.path.exists(f"{self.cloned_repo_directory}/ecosystem.json"):
+            self.logger.info("Configuration file exists.")
+            loaded_config = RepositoryConfiguration.load(
+                f"{self.cloned_repo_directory}/ecosystem.json"
+            )
+            repo_config = cast(PythonRepositoryConfiguration, loaded_config)
+        else:
+            repo_config = PythonRepositoryConfiguration.default()
+
+        # check for tox/.pylintrc/.coveragerc file
+        for i, value in enumerate(file):
+            if os.path.exists(f"{self.cloned_repo_directory}/{value}"):
+                self.logger.info("{value} file exists.")
+                os.rename(
+                    f"{self.cloned_repo_directory}/{value}",
+                    f"{self.cloned_repo_directory}/{file_fault[i]}",
+                )
+
+            # render new tox/.pylintrc/.coveragerc file for tests
+            with open(f"{self.cloned_repo_directory}/{value}", "w") as param_file:
+                if value == "tox.ini":
```

Comment:
So, I correct everything else.

I don't see how you want to create render function like your examples 🤨  
In fact, I have to impression of whatever the implementation at the end, we will have to check the name of the file.

Will we have more template files like them ? Or does these files will be the only ones ? If they ever are the only ones, we can hard coded this part freely.

Reply:
I mean to at least create structure like that
```python
class FileNameConstants:
     TOX_FILE_NAME: str = "tox.ini"
     ...
```

So later everywhere we need it in the code we can reference it using
```python
if file_name == FileNameConstants.TOX_FILE_NAME:
    ...
```

so it will be defined only in one place and that is it :)